src/components/control/autocompleter.jsx

```diff
@@ -88,23 +102,24 @@ const Autocompleter = forwardRef(function Autocompleter(props, ref) {
       listOptionRefs={listOptionRefs}
       name={props.name}
       onBlur={props.onBlur}
-      onChange={props.onChange}
+      onChange={wrappedOnChange}
       onFocus={props.onFocus}
       onInput={onInput}
       onInvalid={props.onInvalid}
       placeholder={props.placeholder}
       readOnly={props.readOnly}
       ref={selectRef}
       required={props.required}
+      specialChildrenRender={!!props.children}
       tooltip={props.tooltip}
       validationMessage={props.validationMessage}
       value={model}
       wrapperRef={props.wrapperRef}
     >
       {({ onSelect }) => (
-        models.map((modelInfo, index) => (
+        renderModels.map((modelInfo, index) => (
           <ListOption key={modelInfo.id} onSelect={onSelect} ref={listOptionRefs[index]} value={modelInfo}>
-            {props.displayValueEvaluator(modelInfo)}
+            { props.children ? props.children(modelInfo) : props.displayValueEvaluator(modelInfo) }
```

Comment:
Is it preferable to use `displayValueEvaluator`?

Reply:
What is nice about this is that your children will be rendered when you select an option.

Reply:
Oh, I see. Just brainstorming here: Looks like we might be able to default `children` as just text value / `<input />` since it seems like a base-case. But let's consider that in a distant future when we understand more use-cases of custom rendering in autocompleters.